### ISSUE TITLE
unity-indicators/indicator-sound-12.10.2_p0_p01_p20160406 config fail

### ISSUE THREAD

**Issue by and3k** [shiznix/unity-gentoo#138]
Hi,

also indicator-sound doesn’t build:

```
 * Package:    unity-indicators/indicator-sound-12.10.2_p0_p01_p20160406
 * Repository: unity-gentoo
 * Maintainer: unity-gentoo overlay
 * Upstream:   https://github.com/shiznix/unity-gentoo/issues
 * USE:        abi_x86_64 amd64 elibc_glibc kernel_linux userland_GNU
 * FEATURES:   preserve-libs sandbox userpriv usersandbox
>>> Unpacking source...
>>> Unpacking indicator-sound_12.10.2+16.04.20160406.orig.tar.gz to /var/tmp/portage/unity-indicators/indicator-sound-12.10.2_p0_p01_p20160406/work
>>> Source unpacked in /var/tmp/portage/unity-indicators/indicator-sound-12.10.2_p0_p01_p20160406/work
>>> Preparing source in /var/tmp/portage/unity-indicators/indicator-sound-12.10.2_p0_p01_p20160406/work ...
>>> Source prepared.
>>> Configuring source in /var/tmp/portage/unity-indicators/indicator-sound-12.10.2_p0_p01_p20160406/work ...
>>> Working in BUILD_DIR: "/var/tmp/portage/unity-indicators/indicator-sound-12.10.2_p0_p01_p20160406/work/indicator-sound-12.10.2_p0_p01_p20160406_build"
cmake -C /var/tmp/portage/unity-indicators/indicator-sound-12.10.2_p0_p01_p20160406/work/indicator-sound-12.10.2_p0_p01_p20160406_build/gentoo_common_config.cmake -G Unix Makefiles -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_INSTALL_LOCALSTATEDIR=/var -DVALA_COMPILER=/usr/bin/valac-0.26 -DVAPI_GEN=/usr/bin/vapigen-0.26 -DCMAKE_BUILD_TYPE=Gentoo -DCMAKE_USER_MAKE_RULES_OVERRIDE=/var/tmp/portage/unity-indicators/indicator-sound-12.10.2_p0_p01_p20160406/work/indicator-sound-12.10.2_p0_p01_p20160406_build/gentoo_rules.cmake -DCMAKE_TOOLCHAIN_FILE=/var/tmp/portage/unity-indicators/indicator-sound-12.10.2_p0_p01_p20160406/work/indicator-sound-12.10.2_p0_p01_p20160406_build/gentoo_toolchain.cmake  /var/tmp/portage/unity-indicators/indicator-sound-12.10.2_p0_p01_p20160406/work
loading initial cache file /var/tmp/portage/unity-indicators/indicator-sound-12.10.2_p0_p01_p20160406/work/indicator-sound-12.10.2_p0_p01_p20160406_build/gentoo_common_config.cmake
-- The C compiler identification is GNU 5.4.0
-- The CXX compiler identification is GNU 5.4.0
-- Check for working C compiler: /usr/bin/x86_64-pc-linux-gnu-gcc
-- Check for working C compiler: /usr/bin/x86_64-pc-linux-gnu-gcc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/x86_64-pc-linux-gnu-g++
-- Check for working CXX compiler: /usr/bin/x86_64-pc-linux-gnu-g++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found PkgConfig: /usr/bin/x86_64-pc-linux-gnu-pkg-config (found version "0.29.1") 
-- Found Vala: /usr/bin/valac-0.26 (found suitable version "0.26.2", minimum required is "0.20") 
-- Checking for modules 'libpulse-mainloop-glib>=0.9.19;glib-2.0>=2.32;gio-unix-2.0>=2.25.13;url-dispatcher-1>=1'
--   Found libpulse-mainloop-glib, version 8.0
--   Found glib-2.0, version 2.48.0
--   Found gio-unix-2.0, version 2.48.0
--   Found url-dispatcher-1, version 1
-- Checking for modules 'gee-0.8;gio-2.0>=2.25.13;gio-unix-2.0;gthread-2.0;libxml-2.0;libnotify;accountsservice'
--   Found gee-0.8, version 0.18.0
--   Found gio-2.0, version 2.48.0
--   Found gio-unix-2.0, version 2.48.0
--   Found gthread-2.0, version 2.48.0
--   Found libxml-2.0, version 2.9.4
--   Found libnotify, version 0.7.6
--   Found accountsservice, version 0.6.40
-- Checking for module 'dbustest-1>=15.04.0'
--   Found dbustest-1, version 15.04.0
-- Checking for module 'gobject-introspection-1.0'
--   Found gobject-introspection-1.0, version 1.46.0
-- GSettings schemas will be installed into /usr/share/glib-2.0/schemas/
-- Checking for module 'libunity-api>=0.1.3'
--   
CMake Error at /usr/share/cmake/Modules/FindPkgConfig.cmake:363 (message):
  A required package was not found
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindPkgConfig.cmake:528 (_pkg_check_modules_internal)
  src/gmenuharness/CMakeLists.txt:1 (pkg_check_modules)
```

libunity is up-to-date (7.1.4_p0_p02_p20151002)

Maybe related to #137 ?

Thanks
Bela

**Comment by shiznix** [comment]
unity-base/unity-api needs to be in DEPEND for unity-indicators/indicator-sound.
Doing a test build run now and commit to follow shortly.

Thanks :)

**Comment by shiznix** [comment]
Should be fixed in commit 6739bdf66ff8e9b0b6947732f7c1b8956519a7db

**Comment by and3k** [comment]
Works, thanks!
